example/swift/.xcode.env.local

```diff
@@ -0,0 +1,2 @@
+export NODE_BINARY=/opt/homebrew/bin/node
```

Comment:
Should this file be committed to the repo?

Reply:
local should stay ignored, this is setup automatically during `pod install`